iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -98,13 +101,21 @@ def handle_regenerate_bundle_request(
         # Pull the from_bundle_image to ensure steps later on don't fail due to registry timeouts
         podman_pull(from_bundle_image_resolved)
 
+        if bundle_replacements:
```

Comment:
Can I ask why do we have this?

Reply:
This should probably check if the `organization` has the `bundle_replacements` customization. If it doesn't and the user has provided `bundle_replacements`, only then we should log this?

Reply:
That was leftover from the API configuration. I don't know if it was a bad commit or rebase. I plan to remove it.